governance/handler.go

```diff
@@ -127,20 +173,43 @@ func (g *Governance) AddVote(key string, val interface{}) bool {
 
 func (g *Governance) adjustValueType(key string, val interface{}) interface{} {
 	k := GovernanceKeyMap[key]
-	reqType := GovernanceItems[k].t
 	var x interface{}
 
 	// When an int value comes from JS console, it comes as a float64
-	if reqType == uint64T && reflect.TypeOf(val) == float64T {
+	if GovernanceItems[k].checkValueType(uint64(0)) && reflect.TypeOf(val) == float64T {
```

Comment:
@yoomee1313 
Is it ok to passing `uint64(0)` to `checkValueType`, not `val` from caller?

Reply:
@aeharvlee sorry for the late! I want to check the `reqType` of the key `k` is uint64T. `val` type is checked at the condition at right.